packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -189,9 +188,12 @@ describe('lion-select-rich', () => {
     expect(el.hasFeedbackFor.includes('error')).to.be.true;
     expect(el.showsFeedbackFor.includes('error')).to.be.false;
 
+    // test submitted prop explicitly, since we dont extend field, we add the prop manually
+    el.submitted = true;
+    await el.updateComplete;
+    expect(el.showsFeedbackFor.includes('error')).to.be.true;
+
     el._listboxNode.children[1].checked = true;
-    // Set touched to true (needed for feedback show) because we simulate a user touching the select
```

Comment:
This code didn't make sense here, probably old, and works without it.

Reply:
Nice